build.gradle

```diff
@@ -32,8 +32,8 @@ subprojects {
         postgresqlDriverVersion = '42.4.1'
         oracleDriverVersion = '19.8.0.0'
         sqlserverDriverVersion = '8.4.1.jre8'
-        grpcVersion = '1.46.0'
-        protobufVersion = '3.20.1'
+        grpcVersion = '1.46.1'
+        protobufVersion = '3.20.3'
```

Comment:
The grpcVersion 1.46.0 was not compatible with the code generated by protobuf version 3.20.3, so it had to be updated to 1.46.1